packages/react-native/React/Views/UIView+React.h

```diff
@@ -83,6 +83,14 @@ typedef struct {
  */
 - (void)reactSetFrame:(CGRect)frame;
 
+
+#if TARGET_OS_OSX // [macOS
+/**
+ * Used by react-native-macos to propagate transform changes internally.
```

Comment:
Maybe just `used on macOS`, since we're in the react-native-macos source? And maybe a small explainer why macOS specifically needs this method while iOS doesn't?